server/bleep/src/semantic.rs

```diff
@@ -505,6 +516,25 @@ impl Semantic {
     }
 }
 
+/// Initialize the `ORT_DYLIB_PATH` variable, consumed by the `ort` crate.
+///
+/// This doesn't do anything on Windows, as tauri on Windows will automatically bundle any `.dll`
+/// files found in the `target/$profile` folder. The `ort` crate by default will also copy the
+/// built dynamic library over to the `target/$profile` folder, when using the download strategy.
+fn init_ort_dylib(dylib_dir: impl AsRef<Path>) {
+    #[cfg(not(windows))]
+    {
+        #[cfg(target_os = "linux")]
+        let lib_name = "libonnxruntime.so";
+        #[cfg(target_os = "macos")]
+        let lib_name = "libonnxruntime.dylib";
+
+        let ort_dylib_path = dylib_dir.as_ref().join(lib_name);
+
+        env::set_var("ORT_DYLIB_PATH", ort_dylib_path);
+    }
+}
+
```

Comment:
It would be good for Docker (for now) if we could just rely on having this as an environment variable, and NOT override it unconditionally. Otherwise we'll need to bump our helm templates too instead of setting it statically in the Docker build step.

This would also be helpful for local work with Nix packaging.

Reply:
Discussed offline: this applies to `--model-dir` as well, so we can take a look at this in another PR.